actions/telemetry_actions.jsx

```diff
@@ -32,14 +32,18 @@ export function shouldTrackPerformance(state = store.getState()) {
 
 export function trackEvent(category, event, props) {
     const state = store.getState();
+
     if (
         isPerformanceDebuggingEnabled(state) &&
         getBool(state, Preferences.CATEGORY_PERFORMANCE_DEBUGGING, Preferences.NAME_DISABLE_TELEMETRY)
     ) {
         return;
     }
 
-    Client4.trackEvent(category, event, props);
+    // Getting the latest user id just in case, we missed earlier.
+    const userId = state.entities.users.currentUserId;
+
+    Client4.trackEvent(category, event, props, userId);
```

Comment:
Should we also do this for `pageVisited`? Even though the frequency of calling that is less, I think it would be similarly affected by missing user ids.

Reply:
+1 - good catch 💯